frontend/package.json

```diff
@@ -38,7 +36,8 @@
     "react-hook-form": "^5.7.2",
     "slate": "^0.57.1",
     "slate-history": "^0.57.1",
-    "slate-react": "^0.57.1"
+    "slate-react": "^0.57.1",
+    "xss": "^1.0.7"
```

Comment:
I replaced `dompurify` with `xss` since `dompurify` isn't isomorphic and requires a window (I kept getting errors in the console when SSR code tried to run `DOMPurify.sanitize`. We could add wrapper code to handle this, but I think it's simpler to just use a library that works out of the box on client or server